universal-login-relayer/src/integration/sql/migrations/20200708000001_create_encrypted_wallets.js

```diff
@@ -0,0 +1,18 @@
+/*
+  THIS IS A MIGRATION
+  This file SHOULD NOT be RENAMED or CHANGED in ANY WAY!
+  If you need to change the database schema create a new migration.
+*/
+
+exports.up = async (knex) => {
+  await knex.schema.createTable('encrypted_wallets', (table) => {
+    table.increments();
+    table.string('walletJSON').notNullable();
```

Comment:
What is the length limit of string in PSQL and what is the length of stringified walletJSON?

Reply:
I think we can say that maximum walletJSON length would be 400. So this is way below length string limit of PSQL which is over 10000000